x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts

```diff
@@ -129,7 +137,7 @@ export const createMetricThresholdExecutor = (libs: InfraBackendLibs) =>
         const timestamp = startedAt.toISOString();
         const actionGroupId = FIRED_ACTIONS.id; // Change this to an Error action group when able
         const reason = buildInvalidQueryAlertReason(params.filterQueryText);
-        const alert = alertFactory(UNGROUPED_FACTORY_KEY, reason);
+        const alert = alertFactory(UNGROUPED_FACTORY_KEY, actionGroupId, reason);
```

Comment:
Is this the correct argument order? Should we type the `actionGroup` argument to `MetricThresholdAlertFactory` more strictly (i.e. as a union of the possible action groups) so the compiler can check this?

Reply:
Whoops... I fixed that and also changed the type so it's more strict.